app/controllers/application_controller.rb

```diff
@@ -29,6 +29,10 @@ def analytics
 
   private
 
+  def decorated_user
+    UserDecorator.new(current_user)
```

Comment:
Is it worth caching this value with ||= ?  I'm not really sure how expensive the constructor is here.